kubernetes/samples/charts/voyager/README.md

```diff
@@ -118,8 +118,66 @@ $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:30305/testwebapp/
 $ curl -k -H 'host: domain2.org' https://${HOSTNAME}:30307/testwebapp/
 ```
 
-## Uninstall the Voyager Operator
-After removing all the Voyager Ingress resources, uninstall the Voyager operator:
+## SSL termination at ingress controller
+This sample demonstrates how to terminate SSL traffic at the ingress controller to access the WebLogic Server Administration Console through the SSL port. 
+
+### 1. Enable "WebLogic Plugin Enabled" on the WebLogic domain level
+
+If you are using WDT to configure the WebLogic domain, you need to add the following resource section at the domain level to the model YAML file.
+```
+resources:
+     WebAppContainer:
+         WeblogicPluginEnabled: true
+```
+If you are using a WLST script to configure the domain, then the following modifications are needed to the respective PY script.
+```
+# Configure the Administration Server
+cd('/Servers/AdminServer')
+set('WeblogicPluginEnabled',true)
+...
+cd('/Clusters/%s' % cluster_name)
+set('WeblogicPluginEnabled',true)
+```
+### 2. Update the frontendRules section ingress resource
+Replace the string `weblogic-domain` with namespace of the WebLogic domain, the string `domain1` with domain UID and the string `adminserver` with name of the Administration Server in the WebLogic domain.
+```
```

Comment:
After finding how to strip the header from Voyager please add the following note
NOTE: If you also have HTTP coming into ingress you will want to make sure that you remove any incoming WL-Proxy-SSL header. This protects you from a malicious user sending in a request and tricking WebLogic into thinking it’s secure when it wasn’t.  Add the following rules in the Voyager proxy configuration to block any WL-Proxy headers coming from the client.

Reply:
done

frontendRules:
  - port: 443
    rules:
    - http-request del-header WL-Proxy-Client-IP
    - http-request del-header WL-Proxy-SSL
    - http-request set-header WL-Proxy-SSL true